dev/Expander/Expander_themeresources.xaml

```diff
@@ -0,0 +1,11 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.UI.Xaml.Controls">
+    
+    <Thickness x:Key="ExpanderExpandChevronMargin">8</Thickness>
+    <StaticResource x:Key="ExpanderForeground" ResourceKey="SystemControlForegroundBaseHighBrush" />
```

Comment:
>ExpanderForeground [](start = 27, length = 18)

Does this need to be a theme resource that changes based on light dark or high constast themes?